src/utils/message_utils.js

```diff
@@ -170,12 +170,7 @@ let MessageUtils = {
   },
 
   loadMessagePackage(msgPackage) {
-    try {
-      messagePackageMap[msgPackage] = ros_msg_utils.Find(msgPackage);
-    }
-    catch (err) {
-      throw new Error(`Unable to include message package ${msgPackage} - ${err}`);
```

Comment:
Wasn't this already re-throwing an error? Is the issue that it was swallowing the original error message?

Reply:
The other function was swallowing the error but this was obscuring the actual error by changing the stack, making it harder for people to debug issues when they occur.

Reply:
:+1: